build.sbt

```diff
@@ -0,0 +1,68 @@
+import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+import uk.gov.hmrc.SbtAutoBuildPlugin
+
+lazy val scoverageSettings = {
+  import scoverage.ScoverageKeys
+  Seq(
+    // Semicolon-separated list of regexs matching classes to exclude
+    ScoverageKeys.coverageExcludedPackages := """uk\.gov\.hmrc\.BuildInfo;.*\.Routes;.*\.RoutesPrefix;.*Filters?;MicroserviceAuditConnector;Module;GraphiteStartUp;.*\.Reverse[^.]*""",
+    ScoverageKeys.coverageMinimum := 80.00,
+    ScoverageKeys.coverageFailOnMinimum := false,
+    ScoverageKeys.coverageHighlighting := true,
+    parallelExecution in Test := false
+  )
+}
+
+lazy val compileDeps = Seq(
+  ws,
+  "uk.gov.hmrc" %% "bootstrap-play-25" % "1.5.0",
+  "uk.gov.hmrc" %% "govuk-template" % "5.19.0",
+  "uk.gov.hmrc" %% "play-ui" % "7.14.0",
+  "uk.gov.hmrc" %% "auth-client" % "2.6.0",
+  "uk.gov.hmrc" %% "play-partials" % "6.1.0",
+  "uk.gov.hmrc" %% "agent-kenshoo-monitoring" % "2.4.0",
+  "uk.gov.hmrc" %% "agent-mtd-identifiers" % "0.10.0",
+  "uk.gov.hmrc" %% "domain" % "5.1.0"
```

Comment:
I think this `uk.gov.hmrc.domain` is a transient dependency brought in by our `agent-mtd-identifiers` library, so can be removed.